docs: improve `Path` documentation (#838)
motion-canvas/motion-canvas: packages/docs/docs/components/path.mdx

});
```

> The `Path` allows us to draw and animate SVG paths.

Reviewer: ```suggestion
The `Path` node allows us to draw and animate SVG paths.
```

:::tip

> Path nodes are great for displaying pre-calculated SVG paths, but the format can be confusing. If you want to have more
> intuitive control of the path, and control each point within Motion Canvas, we reccomend checking out the [`Spline`][spline] component.

Reviewer: ```suggestion
`Path` nodes are great for displaying precalculated SVG paths, but the format can be confusing. If you want to have more
intuitive control of the path, and control each point within Motion Canvas, we recommend checking out the [`Spline`][spline] component.
```

:::

## Defining paths

> We can specify the path to be displayed by setting the data property of the Path component:

Reviewer: ```suggestion
We can specify the path to be displayed by setting the `data` property of the `Path` node:
```

```tsx editor
import {makeScene2D, Path} from '@motion-canvas/2d';
import {createRef} from '@motion-canvas/core';

export default makeScene2D(function* (view) {
  const path = createRef<Path>();

  view.add(
    <Path
      ref={path}
      lineWidth={4}
      stroke={'#e13238'}
      data="M301.113,12.011L576.715,584.766L25.508,584.766L301.113,12.011z"
      position={[-100, -100]}
      scale={0.25}

>     ></Path>,

Reviewer: ```suggestion
    />,
```

});
```

The `Path` node allows us to draw and animate SVG paths.

:::tip

`Path` nodes are great for displaying precalculated SVG paths, but the format
can be confusing. If you want to have more intuitive control of the path, and
control each point within Motion Canvas, we recommend checking out the
[`Spline`][spline] component.

:::

## Defining paths

We can specify the path to be displayed by setting the `data` property of the
`Path` node:

```tsx editor
import {makeScene2D, Path} from '@motion-canvas/2d';
import {createRef} from '@motion-canvas/core';

export default makeScene2D(function* (view) {
  const path = createRef<Path>();

  view.add(
    <Path
      ref={path}
      lineWidth={4}
      stroke={'#e13238'}
      data="M301.113,12.011L576.715,584.766L25.508,584.766L301.113,12.011z"
      position={[-100, -100]}
      scale={0.25}
    />,

>   );

Reviewer: This codeblock contains invalid TS code which is why the verification is not passing.
You're missing the closing brackets:
```suggestion
  );
});
```

});
```

### Animating objects along a path

> Similar to the [`Spline`][spline] and [`Bézier curve`][bezier] components, the `Path` component has
> a `getPointAtPercentage` method that can be used to animate an object along it.

Reviewer: I'd change this to something like:
```suggestion
Like all nodes extending the [`Curve`][curve] class, the `Path` node has
a `getPointAtPercentage` method that can be used to animate an object along it:
```

});
```

### Animating objects along a path

> Like all nodes extending the [`Curve`][curve] class, the `Path` node has a

Reviewer: ``[`Curve`][curve]`` is a [reference-style link](https://www.markdownguide.org/basic-syntax/#reference-style-links), it assumes that the actual link to `curve` is defined somewhere at the bottom of the file:
```md
[curve]: /api/2d/components/Curve
```
